cobra/tests/testdcode.py

```diff
@@ -46,3 +55,48 @@ def test_cobra_dcode(self):
             self.assertEqual(srv.add(), 7)
             self.assertEqual(srv.mult(), 12)
             self.assertEqual(srv.pow(), 81)
+
+
+    def test_dcode_remotecode(self):
+        '''
+        This test can be confusing, because we're using Dcode to access the
+        local machine, not a remote one.
+
+        First, we setup a dcode server with a special base path of "cobra".
+        This makes the contents of Vivisect's `cobra` directory accessible
+        as a top-level.
+
+        Next, we connect to the the Dcode server with the unittest process
+        and import dcode from the "remote" cobra path offered by Dcode.
+        
+        Assertion is just a constant from the remote cobra module
+        '''
+
+        # setup server-side with special path: importing "cobra" as the base,
+        # so anything in that directory becomes top-level, eg. cobra.dcode can
+        # be imported as "import dcode")
+        p = Popen("python3 -m cobra.dcode -P 12347 cobra".split(' '),
+                executable='python3', stdin=PIPE, stderr=PIPE)
+
+        buf = b''
+        while b'Adding path "cobra" to system path' not in buf:
+            buf += p.stderr.read1()
+            logger.debug('waiting... %r', buf)
```

Comment:
I'd prefer the unit tests to only log when things have gone wrong. Our CI tests are already hard enough to parse through as it is, I'd like to at least stem the bleeding.